.github/workflows/main.yml

```diff
@@ -36,13 +42,16 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --progress-bar off --upgrade pip
-          pip install --progress-bar off Django django-guardian
           pip install --progress-bar off pylint==2.6.0 flake8==3.8.2 mypy==0.790 pytest==5.1.3
           pip install --progress-bar off coverage codecov ddt
       - name: Install deep learning frameworks
         run: |
           pip install --progress-bar off torch==${{ matrix.torch-version }}
-          pip install --progress-bar off tensorflow==${{ matrix.tensorflow-version }}
+
+      - name: Pytest
```

Comment:
Now we run pylint, flake8 and mypy everytime we run the pytest. We should still put pytest in its own job to save the time.